wiki/BanchoBot/it.md

```diff
@@ -1,35 +1,159 @@
+
+Se un utente normale invia un comando in un canale di chat pubblico, gli altri utenti non lo vedranno e la risposta verrà mostrata in un messaggio privato con BanchoBot. Gli utenti possono anche utilizzare il comando del client in-game /bb per aprire automaticamente una scheda con BanchoBot e inviare immediatamente il comando.
+
+Di seguito è riportata una lista di tutti i comandi di BanchoBot:
+
+- [Help](#help)
+- [Roll](#roll)
+- [Stats](#stats)
+- [Where](#where)
+- [FAQ](#faq)
+- [Report](#report)
+
+### Help
+
+```
+!help
+```
+
+`!help` mostra la lista di tutti i comandi di BanchoBot. Di seguito un esempio se si usa questo comando.
+
+```
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <user>
+13:00 BanchoBot: STATS <user>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <reason> - call for an admin
+13:00 BanchoBot: REQUEST [list] - shows a random recent mod request
+13:00 BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
+```
+
+<!--note for editors: the code block above reflects the exact response from banchobot -->
+
+*Avviso: Il comando `!request` non è più supportato da BanchoBot.*
+
+### Roll
+
+```
+!roll <argument>/<number>
+```
+
+`!roll` estrae un numero casuale da 1 al numero selezionato. Se il numero non viene specificato o viene fornito un argomento, allora il numero massimo sarà impostato a 100. Esempi di invio di questo comando sono mostrati di seguito:
+
+```
+13:00 pippi: !roll 1000
+13:00 BanchoBot: pippi rolls 109 point(s)
+```
+
+```
+13:01 pippi: !roll probability of failure
+13:01 BanchoBot: pippi rolls 75 point(s)
+```
+
+### Stats
+
+```
+!stats <username>
+```
+
+Il comando `!stats` visualizza le statistiche di gioco dell'utente inserito e il suo stato attuale. L'output dipende dalla [modalità di gioco](/wiki/Game_mode) che l'utente inserito ha giocato per ultimo, anche se BanchoBot non mostrerà da quale modalità di gioco provengono le statistiche. Se viene richiesto di visualizzare le statistiche di un utente che non ha mai giocato a osu!, BanchoBot risponderà con `Utente non trovato`, anche se l'utente esiste. Un esempio di invio di questo comando è mostrato di seguito:
+
+```
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
+```
+
+Ci sono in totale 7 stati che possono essere visualizzati dal comando `!stats` command: Editing, Idle, Lobby, Modding, Multiplayer, Multiplaying and Playing. Un esempio di invio di questo comando per un utente con uno stato disponibile è mostrato di seguito:
```

Comment:
```suggestion
Ci sono in totale 7 stati che possono essere visualizzati dal comando `!stats`: Editing, Idle, Lobby, Modding, Multiplayer, Multiplaying and Playing. Di seguito, un esempio dell'output del comando con utente disponibile:
```
The traduction for `with an available status` I think it is better represented by this form